### DISCUSSION TITLE
Tuya wifi device does not want to update values or updates go into 'sleep'

### DISCUSSION THREAD

**Discussion by AlexK98** [make-all/tuya-local#2590] [Q&A]
Hello,
I'm new to HA and usually use Windows, so, I deployed HassOS 13.2 in VirtualBox under Windows 10.

Added Tuya-Local integration.
Added TOMZN TOMPD-63LW to Tuya-Local using **tompd_63lw_breaker.yaml** configuration file.

Everything looks fine, but we get this:
![2024-12-01 22 04 14](https://github.com/user-attachments/assets/5c145d07-21cb-48c2-978c-9a0babfaac9f)

And it stays like this either until **Refresh sensors** is pressed, or until page of corresponding device is opened in Tuya App or Start Life app (which, i guess, calls some **refresh** method):
![2024-12-01 22 05 14](https://github.com/user-attachments/assets/308a7eda-35c8-4115-b162-162f089b5726)

If I close Tuya / Smart Life app and/or do not press **Refresh sensors** button, the updates stop and freeze showing its last value.
Anyone seen such behavior? And how to fix it?
And this happens not with just this Tomzn device but with other 10 Tuya Wifi devices I have (energy meters, circuit breakers, temp sensors). :(

PS: freezing happens in 1 minute after **refresh**.

**Comment by alivizatos** [comment]
+1 for me. I recently stopped using at all my tuya mobile app, but still freezes. The work around I found was to create an automation to send a command (child lock/unlock) every 5 secs.

**Reply by AlexK98** [reply]
Can you share your automation, please? I'm so new to it. :) A lot of my devices are not sockets and they don't have 'child lock' feature. What would you suggest to trigger for a Circuit Breaker with Metering capability like TOMZN TOMPD63-LW?

**Reply by make-all** [reply]
You have a Refresh sensors button, which is designed for that purpose. So your automation should use that.

**Reply by AlexK98** [reply]
> You have a Refresh sensors button, which is designed for that purpose. So your automation should use that.

Yeah, surprisingly, I just did this automation and it is working, But, for example TOMZN TOMPD-63WIFI does not have such DPid, what would I trigger?
